### DISCUSSION TITLE
Welcome! Introduce yourself

### DISCUSSION THREAD

**Discussion by FrancescoCeruti** [FrancescoCeruti/linux-show-player#269] [Announcements]
👋 Hi everyone!

Welcome to the Linux Show Player discussion section, if you like to introduce yourself this is the right place to start!

I'm Francesco, I write code for a living, I've started Linux Show Player _circa_ summer 2013 to support the theater group I was and still am a part of :heart:

**Comment by fnetX** [comment]
Hi, I am Otto. I'm a student near Berlin, Germany, and I'm a user of Linux Show Player since 2016 when I did some event technology in my past school. By now, I mostly use it within my sustainable event project https://eventbikezero.de/ (we use cargo bikes for all kinds of events, even medium festivals, and even carry stage panels with them), rarely for paid jobs.

I'm currently writing an inventory management system for eventbike_zero (https://codeberg.org/eventbike/inventshare/), but most of my technical contributions go to Codeberg, a non-profit and Free Software alternative to GitHub, where I'm currently the executive director of the backing non-profit association.

I would love to help Linux Show Player with everything except Python. I'm somewhat experienced in all things community management / public relations, QA, and web technology.

**Comment by Mouflon23** [comment]
Hi, I am a french technician in theater. Was searching for a software to use with ubuntu machine. Love the way it look, I'll try it and make feedbacks in here ! Keep doing amazing job ! @FrancescoCeruti

**Comment by Lapazlane** [comment]
Hello, I'm Jon, I'm a writer and performer, currently staging a one man show in the Camden Fringe Festival in London. I've been using Linux since 2007, but I only found LSP a few weeks ago - but it's been an absolute life saver!!! I'm so grateful to Francesco for creating it. Drawing from my current use, I have a few minor suggestions for boosting usability, which I will add in to the New Ideas section.

**Comment by BryanEnneking** [comment]
Hello.  My name is Bryan and I am a software engineer by trade.  In my spare time I am the sound designer for a children's musical theater program that my kids are in.  I started using Linux Show Player in 2021 to control a digital mixer to automate mic cues synchronized with the music.

Even though I only get one to four rehearsals to practice, the shows run very smoothly because all of the music and mic cues are set up ahead of time!

Linux Show Player is a worthy alternative to QLab, and I appreciate the time and effort you and others have put into it.  I hope to make some contributions as well.

**Comment by estudioporta19** [comment]
Hello. My name is Diogo and I am an AV Technician. 
- Studied sound and music production in University
- I love community projects I love Reaper for example, the power of it, is in the tools the community develops.
- Worked in theatre. 
- Working in a Venue that does professional shows, corporate events, amateurish shows, etc etc. 
- I am using Bitfocus Companion to control every equipment and software that has a module for Companion: Art-net (projetors, light-mixers), OSC/UDP (ATEM, BSS Soundweb processors, Audio Mixers, CasparCG, Vmix, OBS, Powerpoint, Reaper, etc. 
- I am using also TheatreMix, Soundplant, Multiplay, ScreenMonkey, TouchOSC, and protocols NDI, OMT.  
- Searched for an alternative to QLab for non Apple computers, found Multiplay in its early years and wasn't good enough.
- Used Soundplant for 2 years found its limitation in external control protocols and cuelist mode is... mehh.
- Programmed https://estudioporta19.github.io/Soundboard/ with AI assistence (since I am not a coder), which didn't failed, but I never tested even in amateur shows since it runs on a web page and I don't know the outcomes. 
- Gave a second chance recently to Multiplay which is working way better and has a good integration with Companion (OSC) and MIDI. 
- Found Linux Show Player last month. It has the two things I like more Cue List (for programmed shows), Cart for last minute shows. Altough I am only testing, since I don't understand how to do some simple stuff yet.

Thank you for your effort!

**Comment by x3paolo-sh** [comment]
Hi everyone!

I’m using a new GitHub account because I recently changed jobs and no longer have access to my previous email address.

I started working as a live sound engineer over 30 years ago, and today I mainly produce theatre shows.

I’ve been a Linux Show Player enthusiast since its early days because, in my opinion, it’s the only true Linux alternative to software like QLab or Arena. I’ve already used LiSP for hundreds of theatre performances, managing music, images, and video cues.

Happy to be here!

**Comment by tobi08151405** [comment]
Hi im Tobias!

I'm a computer science student and a lights and sound tech for theater and bands.
Found LSP while looking for an open replacement for QLab.

Now trying to help expand the feature set.
